src/main/java/com/direwolf20/buildinggadgets/common/schema/BoundingBox.java

```diff
@@ -32,6 +35,30 @@ public BoundingBox(Vec3i min, Vec3i max) {
         this(min.getX(), min.getY(), min.getZ(), max.getX(), max.getY(), max.getZ());
     }
 
+    public int getMinX() {
```

Comment:
Getters below the class methods please.

Reply:
Why?

Reply:
Reference: the entire code base so far.

Reply:
Notice: @MichaelHillcox, @MajorTuvok this belongs to #501

Reply:
Notice: I resolved that over there and am going to pull from v4-1.15 once it's merged, which is why this isn't changed here.